packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -63,8 +63,8 @@ export class OrgList implements vscode.Disposable {
       try {
         const filePath = path.join(
           await ConfigFile.resolveRootFolder(true),
-          '.sf',
-          authFile.username
+          '.sfdx',
```

Comment:
[This](https://github.com/forcedotcom/salesforcedx-vscode/pull/4282/commits/33145b78135034ee1055055733a52b31f8eed42f#diff-2b72c15ab3c2e3d66f269772e80aed9902d2d85d9acb2a462116d03541484d4fR65 ) is where this changed from .sfdx to .sf.  

It seems that this should remain .sfdx I believe, as the [migrating doc](https://github.com/forcedotcom/sfdx-core/blob/v3/MIGRATING_V2-V3.md) has a statement about needing to explicitly point to the .sfdx directory.

Reply:
Tested & verified.  This looks good.

Reply:
Thanks, Jeff!